{{ cookiecutter.project_slug }}/tests/test_package.py

```diff
@@ -0,0 +1,18 @@
+"""Test suite for package-level items."""
+
+from {{ cookiecutter.project_slug }} import __version__
```

Comment:
Should the `__version__` variable also be defined in the template?

Reply:
Oh, yes, it should. Done in https://github.com/Leibniz-HBI/smormlpy/pull/20/commits/a40cca1682d1ffd1520da7bfc21f8a71f715092f.